dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/FileAssetContainerUtil.java

```diff
@@ -165,10 +171,18 @@ public String getContainerIdFromPath(final String fullPath) throws DotDataExcept
         if (null == host) {
 
             try {
-                host = APILocator.getHostAPI().findDefaultHost(APILocator.systemUser(), false);
-            } catch (DotDataException | DotSecurityException e) {
-                host = APILocator.systemHost();
+                host = WebAPILocator.getHostWebAPI()
+                        .getCurrentHost(HttpServletRequestThreadLocal.INSTANCE.getRequest());
```

Comment:
when the path is relative use the current host

Reply:
good